tests/acceptance/testall

```diff
@@ -53,6 +53,11 @@ if [ -f /usr/xpg4/bin/grep ]; then
 fi
 
 #
+# Close stdin file descriptor to avoid tty_interactive check in cf-agent being success.
+#
+exec <&-
```

Comment:
What if we need to use the stdin? Example is ```./testall --gdb``` and I'm not sure if there is more.